x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_charts_container.js

```diff
@@ -112,11 +124,15 @@ function ExplorerChartContainer({ series, severity, tooManyBuckets, wrapLabel, m
               </span>
             )}
             <EuiToolTip position="top" content={textViewButton}>
+              {/* disabling because we need button to behave as link and to have a callback */}
+              {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
               <EuiButtonEmpty
                 iconSide="right"
                 iconType="visLine"
                 size="xs"
-                onClick={redirectToSingleMetricViewer}
+                href={explorerSeriesLink}
+                target={'_blank'}
```

Comment:
not sure if opening in a new tab should be enforced 🤔 cc @peteharverson

Reply:
Now that ctrl+click/right click to open in new tab works, lets default to opening in the same tab.

Reply:
Updated here [`7f82298` (#86182)](https://github.com/elastic/kibana/pull/86182/commits/7f82298865461e3195c50e2d7cc7ac7fb1b1ce69)